Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H

```diff
@@ -248,7 +251,7 @@ void UpdateMomentumPerezElastic (
 
     // Rejection method
     r = amrex::Random(engine);
-    if ( w2 > r*amrex::max(w1, w2) )
+    if ( w2/amrex::max(w1,w2)>r )
```

Comment:
2x: If you keep the term on the RHS, then things should be identical mathematically, but we save division for a faster multiplication.

Also, since `amrex::max(w1,w2)` is used multiple times, consider storing it in a `const` variable?

Reply:
Yes, will modify these later, currently there is still a major issue that has not been solved.